.claude/skills/nextjs-page-migration/references/check-css-vars.sh

```
@@ -0,0 +1,56 @@
+    | sed -E 's/.*"(--[a-zA-Z0-9_-]+)"/\1/' | sort -u)
+fi
+
+# page.module.css 内で定義されているローカル変数を抽出
+local_vars=$(grep -oE '^[[:space:]]*--[a-zA-Z0-9_-]+[[:space:]]*:' "$css_file" \
+  | sed -E 's/^[[:space:]]*(--[a-zA-Z0-9_-]+)[[:space:]]*:/\1/' | sort -u)
+
+# `var(--x, fallback)` は未定義でも fallback に解決されるため安全。
+# フォールバックを持たない参照だけを検査対象にする。
+undefined=0
+for var in $(grep -oE 'var\([[:space:]]*--[a-zA-Z0-9_-]+[[:space:]]*\)' "$css_file" \
+  | sed -E 's/var\([[:space:]]*(--[a-zA-Z0-9_-]+)[[:space:]]*\)/\1/' | sort -u); do
```

Shellcheck (0.11.0)

[info] To read lines rather than words, pipe/redirect to a 'while read' loop.

(SC2013)